packages/terra-worklist-data-grid/src/WorklistDataGrid.jsx

```diff
@@ -320,31 +335,94 @@ function WorklistDataGrid(props) {
   );
 
   const handleClearRowSelection = () => {
+    let message = '';
+    rowSelection.current.rangeUsingLastIndividuallySelectedRow = null;
     if (isAnyRowSelected()) {
-      setAriaLiveMessage(intl.formatMessage({ id: 'Terra.worklist-data-grid.all-rows-unselected' }));
+      message = intl.formatMessage({ id: 'Terra.worklist-data-grid.all-rows-unselected' });
       // Esc (while in row selection mode and rows are selected): Clear selection
       if (onClearSelectedRows) {
         onClearSelectedRows();
       }
     } else if (onDisableSelectableRows) {
-      setAriaLiveMessage(intl.formatMessage({ id: 'Terra.worklist-data-grid.row-selection-mode-disabled' }));
+      rowSelection.current.rangeUsingFocusedRow = null;
```

Comment:
What happens if the consumer doesn't clear the selection?

Reply:
That should be okay because there aren't any rows selected and any further selection will reconfigure the range information. As we discussed yesterday in the meeting with Eric & Chris, the issue around the consumer not honoring the intended action of the user needs more evaluation especially as it relates to aria messages.